TrustModel_Personas.md

```diff
@@ -0,0 +1,195 @@
+  the TEE.
+- Provide the boot images initially required by the TEE during 
+  initialisation or designates a trusted party to do so.
+- Provide the attestation verification service, or designate a trusted party to provide the 
+  attestation verification service.
+
+Example : 3rd party software vendor, cloud solution provider
+
+### Container Image Provider
+
+This persona is responsible for the part of the supply chain that builds container images and 
+provides them for use by the solution. Since a workload can 
+be composed of multiple containers, there may be multiple container image providers, some 
+will be closely connected to the workload provider (business logic containers), others more 
+independent to the workload provider (side car containers). The container image provider is expected
+to use a mechanism to allow provenance of container image to be established when a 
+workload pulls in these images at deployment time. This can take the form of signing or encrypting 
+the container images.
+- Builds container images.
+- Owner of business logic containers. These may contain proprietary algorithms, models or secrets.
+- Signs or encrypts the images.
+- Defines the methods available for verifying the container images to be used.
+- Publishes the signature verification key (public key).
+- Provides any decryption keys through a secure channel (generally to a key management system 
+  controlled by a Key Broker Service).
+- Provides other required verification artifacts (secure channel may be considered).
+- Protects the keys used to sign or encrypt the container images.
+
+It is recognised that hybrid options exist surrounding workload provider and container provider. 
+For example the workload provider may choose to protect their supply chain by 
+signing/encrypting their own container images after following the build patterns already 
+established by the container image provider. 
+
+Example : Sidecar Container Provider
+
+
+### Data Owner
+Owner of data used, and manipulated by the application.
+- Concerned with visibility and integrity of their data.
+- Concerned with compliance and protection of their data.
+- Uses and shares data with solutions. 
+- Wishes to ensure no visibility or manipulation of data is possible by 
+  Orchestration Operator or Cloud Operator personas.
+
+## Conclusion
+
+The key trust relationship here is between the Data Owner and the other personas. The Data Owner 
+trusts the code in the form of container images chosen by the Workload Provider to operate across 
+their data, however they do not trust the Orchestration Operator or Cloud Operator with their 
+data and wish to ensure data confidentiality.
+
+The Workload Provider must not trust the Orchestration Operator or Cloud Operator otherwise 
+they risk losing the trust of the Data Owner. This means the Workload Provider must
+provide assurance to the data owner regarding the integrity of the code and that the data is 
+appropriately protected.
```

Comment:
```suggestion
```
Cut text for conciseness. This concept is somewhat redundant with other content below.